uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manager-leave-calendar.js

```diff
@@ -15,14 +15,17 @@ define([
       return sharedSettings.sharedPathTpl + 'components/manager-leave-calendar.html';
     }],
     controllerAs: 'calendar',
-    controller: ['$controller', '$log', '$q', '$rootScope', 'Calendar', 'Contact', 'OptionGroup', controller]
+    controller: [
+      '$controller', '$log', '$q', '$rootScope', 'shared-settings', 'checkPermissions',
+      'Calendar', 'Contact', 'OptionGroup', controller]
   });
 
-  function controller ($controller, $log, $q, $rootScope, Calendar, Contact, OptionGroup) {
+  function controller ($controller, $log, $q, $rootScope, sharedSettings, checkPermissions, Calendar, Contact, OptionGroup) {
     $log.debug('Component: manager-leave-calendar');
 
     var parentCtrl = $controller('CalendarCtrl');
     var vm = Object.create(parentCtrl);
+    var isAdmin = false;
```

Comment:
Please leave a comment that this var is updated on init, otherwise if you read the code you might consider that this component is not for admin.